components/automate-ui/src/app/pages/team-create/team-create.component.html

```diff
@@ -17,17 +17,23 @@
                 <span class="label">Team {{ versionedNameOrId }}</span>
                 <chef-input ngDefaultControl formControlName="teamId" type="text"></chef-input>
               </label>
-               <chef-error *ngIf="(teamCreateForm.get('teamId').hasError('required') || teamCreateForm.get('teamId').hasError('pattern')) && teamCreateForm.get('teamId').dirty">
-                  A non-blank team {{ versionedNameOrId }} is required
-                </chef-error>
+              <chef-error *ngIf="teamCreateForm.get('teamId').hasError('required') && teamCreateForm.get('teamId').dirty">
+                Team {{ versionedNameOrId }} is required
```

Comment:
so... i think this is fine. but the form labels are _Team ID_ and _Name_ so its a bit weird that the error text uses _Team ID_ and _Team name_ and there is a disconnect there. if we can make those consistent, that would be rad.

Reply:
Fixed it both for team create and team edit. 👍